qiskit/transpiler/passes/mapping/check_map.py

```diff
@@ -13,14 +13,12 @@
 
 from qiskit.transpiler._basepasses import AnalysisPass
 from qiskit.mapper import Layout
-from qiskit.extensions.standard.swap import SwapGate
 
 
 class CheckMap(AnalysisPass):
     """
-    Checks if a DAGCircuit is mapped to `coupling_map`.
-
-    It checks that all 2-qubit interactions are laid out to be physically close.
+    Checks if a DAGCircuit is mapped to `coupling_map` setting `requires_swap` in
```

Comment:
`is_swap_mapped` seems clearer, both in that it's consistent with `is_direction_mapped` and in that it states a property of the dag, rather than a requirement of the dag before it can be run on a device.

Reply:
not sure if I get your point...